modules/end-user-guide/pages/adding-a-vscode-extension.adoc

```diff
@@ -27,42 +37,46 @@ The easiest way to add a Visual Studio Code extension is to add the extension to
     ]
   }
 ----
-. {prod-short} will automatically install the Red Hat Visual Studio Code Java extension in the {prod-short} workspace.
 
-Note: If the specified set of extensions IDs is not available in the {prod-short} registry, the workspace creation will not fail but the extensions will not be available.
+*NOTE* 
+If the specified set of plug-in ids isn't available in the {prod-short} registry, the workspace starts without the plug-in.
 
-The main advantage is that it will work even when not using {prod-short} as primary IDE.
 
 [id="che-theia-plug-ins-YAML"]
-== Using `.che/che-theia-plugins.yaml`
+== Adding plug-in parameters to .che/che-theia-plugins.yaml
+You can add extra parameters to a plug-in by modifying the `.che/che-theia-plugins.yaml` file. These modifications include:
+* Defining the plug-ins for workspace installation.
+* Changing default memory limit.
+* Overriding default preferences.
 
-Relying on `.vscode/extensions.json` is simple but sometimes it requires to specify some extra parameters for extensions.
 
-Here comes the optional `.che/che-theia-plugins.yaml` file definition
+=== Defining the plug-ins for workspace installation
+Define the plug-ins to be installed in the workspace.
 
-=== Defines the extensions to install in the workspace
+.Prerequisites
+. You have the `.che/che-theia-plugins.yaml` file in the root of the GitHub repository.
 
 .Procedure
-
-. Add `.che/che-theia-plugins.yaml` file if it is missing at the root of the Git repository.
-
-. Add the id of the extension using the '/' separated format.
+. Add the id of the plug-in in the  `.che/che-theia-plugins.yaml` file. Use the `/` sign to separate the publisher and plug-in name:
```

Comment:
```suggestion
. Add the `id` of the plug-in in the  `.che/che-theia-plugins.yaml` file. Use the `/` sign to separate the publisher and plug-in name:
```
@benoitf, let's use 'extension' in the context of VS Code and 'plug-in' in the context of Che. In this sentence particularly, it would seem strange to be adding an 'extension' into the `che-theia-plugins.yaml` file :)

Reply:
ok 👍